src/seedu/addressbook/ui/Tagging.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.ui;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.